src/models/units.ts

```diff
@@ -154,6 +197,31 @@ export const units: Units = {
         phenotypeHeading: "Pea type",
         getPhenotypeLabel: (phenotype) => phenotype.charAt(0).toUpperCase() + phenotype.slice(1),
         getGenotypeHTMLLabel: (genotype) => genotype,
+        chartTypes: {
+          phenotype: {legend: [
+            {label: "Smooth", color: colors.colorDataMouseBrownLightRep},
+            {label: "Wrinkled", color: colors.colorDataMouseBrownMediumRep}
+          ], title: "Pea Types" },
+          genotype: {legend: [
+            {label: "RR Peas", color: colors.colorDataMouseBrownLightRep},
+            {label: "Rr Peas", color: colors.colorDataMouseBrownMediumRep},
+            {label: "rR Peas", color: colors.colorDataMouseBrownMediumRep},
+            {label: "rr Peas", color: colors.colorDataMouseBrownDarkRep}
+          ], title: "Genotypes" }
+        },
+        getChartData: (chartType, data) => {
+          let pieData = [];
+          if (chartType === "genotype") {
+            pieData = [{label: "RR", value: data.RR, color: colors.colorDataMouseBrownLightRep},
+                       {label: "Rr", value: data.Rr, color: colors.colorDataMouseBrownMediumRep},
+                       {label: "rR", value: data.rR, color: colors.colorDataMouseBrownMediumRep},
+                       {label: "rr", value: data.rr, color: colors.colorDataMouseBrownDarkRep}];
+          } else {
+            pieData = [{label: "Smooth", value: data.smooth, color: colors.colorDataMouseBrownLightRep},
+                       {label: "Wrinkled", value: data.wrinkled, color: colors.colorDataMouseBrownMediumRep}];
```

Comment:
Same with the colors in `getChartData` - we don't have a pea spec yet to set these colors, right?